inst/extdata/github-repos.Rmd

```diff
@@ -0,0 +1,99 @@
+---
+title: "GitHub repo details"
+output:
+  html_document:
+    code_folding: hide
+    toc: true
+    theme: united
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk[["set"]](
+  echo = TRUE,
+  cache = TRUE
+)
+```
+
+```{r get_repos}
+
+org <- "RMI-PACTA"
+all_repos <- get_repos(org)
+
+```
+
+```{r active_repos}
+active_repos <- all_repos |>
+  lapply(
+    FUN = function(x) {
+      if (x[["archived"]] || x[["disabled"]]) {
+        return(NULL)
+      } else {
+        return(x)
+      }
+    }
+  )
+#remove null elements
+active_repos <- active_repos[!vapply(
+  X = active_repos,
+  FUN = is.null,
+  FUN.VALUE = logical(1L)
+)]
+```
+
+
+```{r identify_prod_dependencies, include=FALSE}
+
+prod_checked <- lapply(active_repos, prod_checks)
```

Comment:
When I try to run this, it errors out for me, weirdly on the `pacta.sit.rep` repo. 

Repro steps: 
* Source this `.Rmd`

When I ran it, it was the 62th repo that caused the failure: 
``` r
prod_checks(active_repos[[62]])
```

Reply:
@jdhoffa That's because the CODEOWNERS file for this repo is in the wrong location (on `main`). I've added a fix in [abe689b](https://github.com/RMI-PACTA/pacta.sit.rep/pull/13/commits/abe689b43d2776810e5fdd400fa1f42535860830), but that won't resolve until it's been merged into `main`

Reply:
Ahaha good catch